handyrl/envs/geister.py

```diff
@@ -150,7 +144,7 @@ def __init__(self):
         self.head_v = ScalarHead((filters * 2, 6, 6), 1, 1)
         self.head_r = ScalarHead((filters * 2, 6, 6), 1, 1)
 
-    def init_hidden(self, batch_size=None):
+    def init_hidden(self, batch_size=[]):
```

Comment:
IMHO, I prefer `batch_size=None` like [this](https://stackoverflow.com/a/366430). What do you think?
Also, https://github.com/DeNA/HandyRL/blob/master/handyrl/evaluation.py#L79

Reply:
I changed this to show that we don't need `if` branch, but since this is a user-defined location, None is also fine.

Reply:
I don't think it needs to be the default argument.

Reply:
👍